lib/timescaledb/stats/hypertables.rb

```diff
@@ -0,0 +1,86 @@
+require 'bytesize'
+
+require_relative './chunks'
+
+module Timescaledb
+  class Stats
+    class Hypertables
+      # @param [Timescaledb:Connection] connection The PG connection.
+      # @param [Array<String>] hypertables The list of hypertable names.
+      def initialize(connection, hypertables = [])
+        @connection = connection
+        @hypertables = hypertables
+      end
+
+      delegate :query, :query_first, :query_count, to: :@connection
+
+      # @return [Hash] The hypertables stats
+      def to_h
+        {
+          count: @hypertables.count,
+          uncompressed_count: uncompressed_count,
+          approximate_row_count: approximate_row_count,
+          chunks: Timescaledb::Stats::Chunks.new(@connection, @hypertables).to_h,
+          size: size
+        }
+      end
+
+      private
+
+      def uncompressed_count
+        @hypertables.count do |hypertable|
+          query("SELECT * from hypertable_compression_stats('#{hypertable}')").empty?
+        end
+      end
+
+      def approximate_row_count
+        @hypertables.each_with_object(Hash.new) do |hypertable, hsh|
+          row_count = query_first("SELECT * FROM approximate_row_count('#{hypertable}')").approximate_row_count.to_i
+
+          hsh[hypertable.to_sym] = row_count
+        end
```

Comment:
Can we use a meaningful name instead of hsh?

Suggestions: `summary`, `count`, `resume`.

Reply:
Makes sense, will change it 👍